globals.go

```diff
@@ -231,6 +231,16 @@ const (
 	DefaultShutdownWaitSeconds = 10
 )
 
+var TaskFailureStatuses []string = []string{
+	TaskTimedOut,
```

Comment:
I noticed this differs from `IsFailedTaskStatus` only by `TaskTimedOut`. I think these 2 can be unified, and `IsFailedTaskStatus` is just missing `TaskTimedOut`.

Reply:
`IsFailedTaskStatus` does not exist

Reply:
ah, thought you were referring to code i had written.  i do not use `IsFailedTaskStatus` in my code but i refactored the function to use `TaskFailureStatuses`